web/src/features/hfiCalculator/slices/hfiCalculatorSlice.ts

```diff
@@ -137,6 +145,10 @@ const dailiesSlice = createSlice({
       state.loading = false
       state.changeSaved = false
     },
+    fetchFuelTypesFailed(state: HFICalculatorState, action: PayloadAction<string>) {
+      state.error = action.payload
```

Comment:
What happens when station load fails, but fuel type load works. Does the error message stick around? Does the loading indicator stop when one of those calls comes back? (Should be in loading state until both responses have come back).